src/renderers/dom/ReactDOMStringRenderer.js

```diff
@@ -36,10 +40,13 @@ function renderToString(element) {
  * See https://facebook.github.io/react/docs/react-dom-server.html#rendertostaticmarkup
  */
 function renderToStaticMarkup(element) {
-  invariant(
-    React.isValidElement(element),
-    'renderToStaticMarkup(): You must pass a valid ReactElement.',
-  );
+  const disableNewFiberFeatures = ReactFeatureFlags.disableNewFiberFeatures;
+  if (disableNewFiberFeatures) {
+    invariant(
```

Comment:
:smiley: I understand your mean, I fell that children do not need validation. because when [resolve](https://github.com/gzhappysky/react/blob/02101d5d4acccce66b303535aeea5bb07d9a384f/src/renderers/shared/server/ReactPartialRenderer.js#L309) method meet array type children , It will return to [render](https://github.com/gzhappysky/react/blob/02101d5d4acccce66b303535aeea5bb07d9a384f/src/renderers/shared/server/ReactPartialRenderer.js#L485) method, the render method will check if child is a array type, if it is then push to stack,  so resolve only validates single element.